react/components/ProductPriceTotal.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react"
+import { Spinner } from 'vtex.styleguide'
+import useQueryGetProductPrice from '../hooks/actions/useQueryGetProductPrice'
+
+import styles from './productPriceTotal.css'
+
+const ProductPriceTotal = ({ skuReference = 0, productQuantity = 0, currency = '$' }) => {
+
+  const { refetch, data, error: productError, loading } = useQueryGetProductPrice('reference', skuReference)
+
+  useEffect(() => {
+    refetch()
+  }, [productQuantity])
+
+  const price = (data?.product.items ?? []).find((item) => item)?.sellers?.find((seller) => seller)?.commertialOffer?.Price;
+
+  const handlePrice = () => {
+    if (productQuantity && price) {
+      const lineTotal = productQuantity * price
+      return `${currency} ${price ? parseFloat(lineTotal).toFixed(2) : null}`
```

Comment:
we should format price using react intl. not like this.

Reply:
Have an issue with doing this change.